.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "blt"]
 	path = blt
 	url = https://github.com/LLNL/blt.git
+[submodule ".radiuss-ci"]
+	path = .radiuss-ci
+	url = git@github.com:LLNL/radiuss-ci.git
```

Comment:
Should this be https://github.com/LLNL/radiuss-ci.git ?  The Travis builds fail due to failure to clone the submodule.

Reply:
Yes, good catch!